### PR TITLE
Beautified profile view

### PR DESCRIPTION
**Pull request by friedow**

### CODE REVIEW

File: src/tetris/ui/dialog/evaluationView.ts [hunk 1]
```diff
@@ -85,12 +84,17 @@ export default class EvaluationView {
 	}
 
 	private _displayLastPlayerImage(): void {
-		if (this.profile.image) {
-			this._pictureContainer.classList.add("display");
-			this._pictureContainer.src = BASE64_IMAGE_PREFIX + this.profile.image;
-		} else {
-			this._pictureContainer.classList.remove("display");
-		}
+		// for (const [index, fppAnalysis] of this.profile.fppFaceAnalysis.valueHistory.entries()) {
```
Comment (Johannes-Schneider) [inline src/tetris/ui/dialog/evaluationView.ts]: If you dont intend to use this code anymore, delete it.

**Comment by julianweise** [comment]
screen shot 2018-09-07 at 5 27 08 pm

Is it intended not to be of full height?

**Comment by friedow** [comment]
I changed the style according to your suggestions @julianweise. Please have another look.

**Comment by friedow** [comment]
I could not reproduce your problem of the site not being full height. Even if there is no data, the orange background should be full height.
image

File: assets/css/dialog.css [hunk 2]
```diff
@@ -240,8 +242,15 @@ input[type="button"].dialog-reject-element:hover, button.dialog-reject-element:h
 #evaluation-dialog {
     max-height: 100%;
     height: unset;
```
Comment (julianweise) [inline assets/css/dialog.css]: That's the problem of the height not being 100%

File: assets/css/timeline.css [hunk 3]
```diff
@@ -26,6 +26,8 @@
 #evaluation-dialog-timeline {
     margin-top: 80px;
     margin-bottom: 300px;
+    margin-left: -1px;
+    margin-right: -1px;
```
Comment (julianweise) [inline assets/css/timeline.css]: We should have at least a padding left and right of about 20px.

**Pull request merged**